### PR TITLE
Codify time an engineer is allowed to work on OSS

### PR DESCRIPTION
**Pull request by christian-bromann**
After our recent engineering all hands this question was asked and I would like to codify this into our guidelines so everyone is one the same page.

### CODE REVIEW

**Review by diemol — DISMISSED** [review]

**Comment by christian-bromann** [comment]
Previous version was:

> Participation in open source projects is highly encouraged and part of every engineers job at Sauce Labs. While you are working with your team on a product backlog you should always ensure to take some time off and spend it to expand your capabilities and experiences working on an Open Source project of your choice. If your backlog doesn't allow this, please talk to your manager.

> The Open Source Program Office wants to make this process as simple as possible. All Saucers are expected to follow common sense regarding the type of information and code is contributed to upstream open source projects. If you are in doubt, please reach out to the [OSPO](mailto:opensource@saucelabs.com) or ask your manager for direction.

File: content/docs/contributing.md [hunk 1]
```diff
@@ -4,12 +4,15 @@ description: "Contributing to Open Source"
 draft: false
 ---
 
-Participation in open source projects is highly encouraged, and the Open Source Program Office wants to make this process as simple as possible. All Saucers are expected to follow common sense regarding the type of information and code is contributed to upstream open source projects. If you are in doubt, please reach out to the OSPO or ask your manager for direction.
+Participation in open source projects is highly encouraged and is an essential part of the engineering culture at Sauce Labs. It can be an excellent tool to learn new technologies and professionally grow, which is a responsibility of every engineer. If your backlog doesn't allow for this, please talk to your manager.
```
Comment (diemol) [inline content/docs/contributing.md]:
```suggestion
Participation in open source projects is highly encouraged and is an essential part of the engineering culture at Sauce Labs. We invite all engineers at Sauce Labs to use open source as an opportunity to learn new technologies, professionally grow, and contribute to the open source ecosystem. If you are struggling to find time for open source during your work time, please reach out to the [OSPO](mailto:opensource@saucelabs.com) or to your manager.
```

Comment (christian-bromann) [reply content/docs/contributing.md]: 👍

**Review by diemol — DISMISSED** [review]

**Comment by boneskull** [comment]
I've rebased this onto `dev`--otherwise I think this can be merged.  @diemol ?

**Review by boneskull — APPROVED** [review]

**Pull request closed without merging**